eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -158,6 +162,11 @@ jobs:
     - name: sourcesPath
       value: $(vmrPath)
 
+  # Must be a path under $(sourcesPath). Inside the docker container, we mount $(sourcesPath) to /vmr
+  # and can't write outside of that folder.
+  - name: artifactsStagingDir
+    value: $(sourcesPath)/artifacts/staging
```

Comment:
If we weren't running our own custom docker process, could we default this to AzDO's Artifacts.StagingDirectory?

Reply:
Yes